src/area_selector/mod.rs

```diff
@@ -160,6 +160,7 @@ impl AreaSelector {
 
             let monitor_geometry = RootExt::display(transient_for)
                 .monitor_at_surface(&transient_for.surface())
+                .expect("No monitor found")
```

Comment:
This could be the following instead, so it won't panic:
```suggestion
                .context("No monitor found")?
```